### DISCUSSION TITLE
Comparison to nextcloud repod?

### DISCUSSION THREAD

**Discussion by sttrebo** [SamTV12345/PodFetch#1080] [General]
I've got an instance if Nextcloud that I'm trying to spin down. One of the remaining tasks I still use it for is podcasts (Repod).  I've got Podfetch up and running and it looks that it has about the same capabilities?  Anyone here know if I may be missing anything?  Podfetch/Antennapod so far seems to work great.  Plus it looks like its a lot easier on the CPU resources than the nextcloud option?

Thanks

**Comment by SamTV12345** [comment]
Yes it's nearly identical to RePod. It's a lot easier to setup than a complete Nextcloud instance just for podcasts. You can set it up even on a RPI 1 . If it runs Linux you can run it for sure :)

**Reply by sttrebo** [reply]
Thanks for the info.  I'm running it on my synology nas using this [https://mariushosting.com/how-to-install-podfetch-on-your-synology-nas/](url).  Runs pretty smooth, haven't noticed any problems.  Plus the interface is much nicer...
